doc/config.md

```diff
@@ -300,6 +300,97 @@ updated and how often aggregated statistics are written to the log
 
 Default: 60
 
+### prepared_statement_cache_size
```

Comment:
`prepared_statement_cache_size` is a long parameter name. When you say "size" my first impression is that the unit is in bytes, MB, GB. However, it is the maximum number of prepared statements to keep on a server connection. Do we need "cache" and/or "size" in the parameter name? Maybe a generic name is good enough if the documentation explains what it is for in the first 2 sentences. By generic name I mean `prepared_statements` or `prepared_queries`. I thought about `cached_plans` but it seems too technical.

Reply:
Good point on the `size` suffix suggesting a byte unit. I think `prepared_statements` is a bit to generic though. How about `max_prepared_statements`?

Reply:
Works for me.